cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -150,7 +155,27 @@ def push_to_remote(self, base_branch, head_branch):
         except subprocess.CalledProcessError:
             click.echo(f"Failed to push to {self.pr_remote} \u2639")
         else:
-            self.open_pr(self.get_pr_url(base_branch, head_branch))
+            if os.getenv("GH_AUTH"):
+                self.create_gh_pr(base_branch, head_branch, commit_message)
+            else:
+                self.open_pr(self.get_pr_url(base_branch, head_branch))
+
+    def create_gh_pr(self, base_branch, head_branch, commit_message):
```

Comment:
If you're already checking for the envvar above, would it make sense to make it a required argument and just pass it in explicitly?

Reply:
Ok, I made the token a required argument.